index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+<meta charset="UTF-8">
+<meta http-equiv="X-UA-Compatible" content="IE=edge">
+<meta name="viewport" content="width=device-width, initial-scale=1.0">
+<title>Hello, Welome!</title>
+<link rel="stylesheet" href="style.css">
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+
+</head>
+
+<body>
+   <navbar class="navbar-fixed-top">
+<p>Welcome! <img src="/home/edwin/Portfolio-webpage/Images/Capacity.jpg"> <img src="/home/edwin/Portfolio-webpage/Images/Icon - Menu.png"> <img src="/home/edwin/Portfolio-webpage/Images/Wifi.png"> </p>
+   </navbar>
```

Comment:
- Good job for trying to use semantic tags, I think it is better to use `nav` instead of `navbar` and make the `nav` inside the `header ` 👌👍
- Kindly, use `a` tag for your logo because of convention, semantics, and user experience(UX). [Check this link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more info. 👌
- Please be sure that the image you are using is available in the repository and used the right `path`.👌

Reply:
Hi Binhussen, thank you for taking your time to review the PR. However, my template doesn't include a logo. For this reason, I fail to understand your second point. Kindly rephrase.